app/src/main/java/com/project/technion/appark/Maps/GoogleMapsActivity.java

```diff
@@ -205,19 +231,43 @@ public void onDataChange(DataSnapshot dataSnapshot) {
                     marker.setTag(offer);*/
 
 
-                    IconGenerator icg = new IconGenerator(GoogleMapsActivity.this);
-                    String currency = Constants.CURRENCY;
-                    if (offer.price > 10) currency += Constants.CURRENCY;
```

Comment:
You should use Constants class instead of an hardcoded "$".
Now the currency is NIS